server/channels/api4/channel_test.go

```diff
@@ -4074,37 +4212,44 @@ func TestAddChannelMember(t *testing.T) {
 
 	otherUser := th.CreateUser()
 	otherChannel := th.CreatePublicChannel()
-	client.Logout(context.Background())
-	client.Login(context.Background(), user2.Id, user2.Password)
+
+	_, err = client.Logout(context.Background())
+	require.NoError(t, err)
+	_, _, err = client.Login(context.Background(), user2.Email, user2.Password)
```

Comment:
Another broken `Login` call that leads to invalid status checks.

Reply:
We are just adding the error checking, but changing the status code checks. How is that making the difference? Could you clarify?

Reply:
The fix is
```diff
- client.Login(context.Background(), user2.Id, user2.Password)
+ client.Login(context.Background(), user2.Email, user2.Password)
```

Reply:
Ahh sorry!